metrics/size.table/table.go

```diff
@@ -0,0 +1,166 @@
+// Copyright 2022 Block, Inc.
+
+package sizetable
+
+import (
+	"context"
+	"database/sql"
+
+	"github.com/cashapp/blip"
+	"github.com/cashapp/blip/sqlutil"
+)
+
+const (
+	DOMAIN = "size.table"
+
+	opt_total         = "total"
+	OPT_EXCLUDE       = "exclude"
+	OPT_INCLUDE       = "include"
+	OPT_SCHEMA_FILTER = "schema"
```

Comment:
Dead code.

Reply:
I don't see where `OPT_SCHEMA_FILTER` is used?

Reply:
sorry I don't think I pushed the changes